src/geometric_algebra/nursery/graded.lean

```diff
@@ -106,10 +121,14 @@ namespace graded_module
   end
   
   /-- Chisholm 6b -/
-  lemma grade_of_sum (r : ℤ) (a b : G) : ⟨a + b⟩_r = ⟨a⟩_r + ⟨b⟩_r := by simp
+  lemma grade_of_sum (r : ℤ) (a b : G) :
+  ⟨a + b⟩_r = ⟨a⟩_r + ⟨b⟩_r :=
+  by simp only [dfinsupp.add_apply, linear_map.map_add]
```

Comment:
I'm not convinced this type of change is worthwhile - terminal simp is considered fine, right?

I don't mind you merging this change, but it would be nice to learn whether this is the mathlib way.

Reply:
Yes, it's fine. But squeezing it to `simp only` makes the rewrite rules involved clear, stablize the behavior and make compiling fast when it scales.